pkg/gui/app_status_manager.go

```diff
@@ -2,43 +2,59 @@ package gui
 
 import "github.com/jesseduffield/lazygit/pkg/utils"
 
+// appStatus is used to store information about the status
+// of the current session
 type appStatus struct {
 	name       string
 	statusType string
 	duration   int
 }
 
+// statusManager
 type statusManager struct {
 	statuses []appStatus
 }
 
+// removeStatus removes the status from the application.
+// name: the name of the status
 func (m *statusManager) removeStatus(name string) {
-	newStatuses := []appStatus{}
-	for _, status := range m.statuses {
+
```

Comment:
I think whitespace should only be used to separate logically distinct blocks of code. I don't think it should be used at the top or bottom of a function's body. Is this a convention used in other popular repos?

Reply:
It's not a convention, it's more that i've gotten used to it since some C formats, was just checking out how it looks.

Reply:
Fair enough. I reckon we should stick to only using blank spaces for separating chunks